docs/getting-started-guides/mesos.md

```diff
@@ -0,0 +1,297 @@
+$ ./bin/km controller-manager \
+  --master=$servicehost:8888 \
+  --mesos_master=${mesos_master} \
+  --v=1 >controller.log 2>&1 &
+
+$ ./bin/km scheduler \
+  --address=${servicehost} \
+  --mesos_master=${mesos_master} \
+  --etcd_servers=http://${servicehost}:4001 \
+  --mesos_user=root \
+  --api_servers=$servicehost:8888 \
+  --v=2 >scheduler.log 2>&1 &
+```
+
+Start the service proxy on each Mesos slave node:
+
+```bash
+$ sudo ./bin/km proxy \
+  --bind_address=${servicehost} \
+  --etcd_servers=http://${servicehost}:4001 \
+  --logtostderr=true >proxy.log 2>&1 &
+```
+
+Disown your background jobs so that they'll stay running if you log out.
+
+```bash
+$ disown -a
+```
+#### Validate KM Services
+Interact with the kubernetes-mesos framework via `kubectl`:
+
+```bash
+$ bin/kubectl get pods
+POD        IP        CONTAINER(S)        IMAGE(S)        HOST        LABELS        STATUS
+```
+
+```bash
+$ bin/kubectl get services       # your service IPs will likely differ
+NAME            LABELS                                    SELECTOR            IP             PORT
+kubernetes      component=apiserver,provider=kubernetes   <none>              10.10.10.2     443
+kubernetes-ro   component=apiserver,provider=kubernetes   <none>              10.10.10.1     80
+```
+Lastly, use the Mesos CLI tool to validate the Kubernetes scheduler framework has been registered and running:
+```bash
+$ mesos state | grep "Kubernetes"
+         "name": "Kubernetes",
+```
+Or, open the Mesos web GUI and look for the registered framework named "Kubernetes".
+
+## Spin up a pod
+
+Write a JSON pod description to a local file:
+
+```bash
+$ cat <<EOPOD >nginx.json
+{ "kind": "Pod",
+"apiVersion": "v1beta1",
+"id": "nginx-id-01",
+"desiredState": {
+  "manifest": {
+    "version": "v1beta1",
+    "containers": [{
+      "name": "nginx-01",
+      "image": "dockerfile/nginx",
+      "ports": [{
+        "containerPort": 80,
+        "hostPort": 31000
+      }],
+      "livenessProbe": {
+        "enabled": true,
+        "type": "http",
+        "initialDelaySeconds": 30,
+        "httpGet": {
+          "path": "/index.html",
+          "port": "8081"
+        }
+      }
+    }]
+  }
+},
+"labels": {
+  "name": "foo"
+} }
+EOPOD
+```
+
+Send the pod description to Kubernetes using the `kubectl` CLI:
+
+```bash
+$ bin/kubectl create -f nginx.json
+nginx-id-01
+```
+
+Wait a minute or two while `dockerd` downloads the image layers from the internet.
+We can use the `kubectl` interface to monitor the status of our pod:
+
+```bash
+$ bin/kubectl get pods
+POD          IP           CONTAINER(S)  IMAGE(S)          HOST                       LABELS                STATUS
+nginx-id-01  172.17.5.27  nginx-01      dockerfile/nginx  10.72.72.178/10.72.72.178  cluster=gce,name=foo  Running
+```
+
+Verify that the pod task is running in the Mesos web console.  Click on the Kubernetes framework.  The next screen should show the running Mesos task that started the Kubernetes pod.
```

Comment:
Explain how to access the Mesos web console? (Ignore if it's the same thing as "open the Mesos web GUI" which I commented on earlier)